tests/Type/DatePickerTypeTest.php

```diff
@@ -30,7 +30,7 @@ final class DatePickerTypeTest extends TypeTestCase
     /**
      * @var Stub&TranslatorInterface
      */
-    private $translator;
+    private TranslatorInterface $translator;
```

Comment:
so, you can type this with TranslatorInterface even if it is a Stub?

Reply:
Yes, since the stub implements the interface. Or it wouldnt be accepted as constructor argument.